### PR TITLE
success timeout

### PR DESCRIPTION
**Pull request by AdiZav**
Move success timeout param to allow usage with `with` keyword

### CODE REVIEW

**Review by rnetser — APPROVED** [review]

File: ocp_resources/node_network_configuration_policy.py [hunk 1]
```diff
@@ -103,6 +104,7 @@ def __init__(
         self.state = state or self.Interface.State.UP
         self.set_ipv4 = set_ipv4
         self.set_ipv6 = set_ipv6
+        self.success_timeout = success_timeout
```
Comment (myakove) [inline ocp_resources/node_network_configuration_policy.py:107]: Please remove it from here, the function `wait_for_status_success` should get it.

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/node_network_configuration_policy.py (hunk 1 above)
Comment (AdiZav) [reply ocp_resources/node_network_configuration_policy.py:107]: Can't supply it `to wait_for_status_success` if an nncp is deployed using `with`, unless passed on __init__

**Review by myakove — DISMISSED** [review]

**Comment by myakove** [comment]
@AdiZav please rebase and verifiy

**Comment by AdiZav** [comment]
!verified

**Review by myakove — APPROVED** [review]

**Pull request merged**